src/styles/nested-list.scss

```diff
@@ -5,378 +5,253 @@ $block: #{$fd-namespace}-nested-list;
-          background: var(--sapList_Active_Background);
-        }
+        @include fd-nested-list-states-colors(var(--sapList_Active_TextColor), var(--sapList_Active_Background));
       }
 
       .#{$block}__title,
       .#{$block}__icon,
-      .#{$block}__button {
+      .#{$block}__arrow {
         color: var(--sapList_Active_TextColor);
       }
     }
-
-    &.has-child {
-      justify-content: space-between;
-      padding: 0;
-
-      > .#{$block}__link,
-      > .#{$block}__item {
-        border: none;
-      }
-
-      > .#{$block}__link {
-        @include fd-fiori-focus();
-
-        background: transparent;
-        border: none;
-        height: 100%;
-
-        > .#{$block}__title {
-          padding-right: 0;
-
-          @include fd-rtl() {
-            padding-left: 0;
-          }
-        }
-      }
-    }
-  }
-
-  &__link {
-    cursor: pointer;
   }
 
-  &__button.#{$button} {
-    @include set-height(100%);
-
-    border: none;
-    outline: none;
-    background: none;
-    min-width: 2.5rem;
-    text-decoration: none;
-    color: var(--sapContent_IconColor);
+  /* This wrapper is needed because we can't apply overflow: hidden on fd-nested-list__link as the focus outline is cut.
+  And we need overflow hidden for the selected indicator on the left side which needs to follow the item border radius. */
+  &__link-container {
+    @include fd-reset();
 
-    @include fd-icon-selector() {
-      font-size: var(--sapFontLargeSize);
+    @include fd-flex() {
+      align-items: center;
     }
+
+    width: 100%;
+    height: 100%;
+    overflow: hidden;
+    position: relative;
+    border-radius: $fd-nested-list-item-border-radius;
   }
 
   &__icon {
     @include fd-icon-element-base() {
       @include fd-flex-center();
+      @include set-width(3rem);
+      @include set-height(100%);
 
-      height: 100%;
-      min-width: 2.75rem;
-      color: var(--sapList_TextColor);
-      font-size: var(--sapFontHeader4Size);
+      color: currentColor;
+      font-size: var(--sapFontSize);
     }
   }
 
   &__title {
     @include fd-reset();
+    @include fd-ellipsis();
 
-    height: 100%;
     width: 100%;
-    font-size: var(--sapFontLargeSize);
-    color: var(--sapList_TextColor);
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
-    padding-right: 1rem;
-    display: flex;
-    align-items: center;
-
-    &:first-child {
-      padding-left: 2.75rem;
-    }
+    text-align: left;
+    color: currentColor;
+    display: inline-block;
+    font-size: var(--sapFontSize);
+    height: $fd-nested-list-item-height;
+    line-height: $fd-nested-list-item-height;
 
     @include fd-rtl() {
       text-align: right;
-      padding-right: 0;
-      padding-left: 1rem;
-
-      &:first-child {
-        padding-right: 2.75rem;
-        padding-left: 0;
-      }
     }
   }
 
-  &__group-header {
+  &__arrow {
     @include fd-reset();
+    @include fd-flex-center();
+    @include set-width(1rem);
```

Comment:
prefix?

Reply:
I opened a PR for renaming the mixins missing the prefix. Depending on which PR is merged first I will rebase and update. 
https://github.com/SAP/fundamental-styles/pull/3921